README.md

```diff
@@ -14,42 +14,82 @@ We use this prototype internally to LeanIX to generate SBOMs for our projects. I
 
 1. Run the Docker container from the latest image:
 
+GitHub:
 ```console
-docker run --pull=always \
+docker run --pull=always --rm \
            -v /var/run/docker.sock:/var/run/docker.sock \
            -v <temp-folder-to-be-used-for-storing-data>:/tempDir \
            -e MOUNTED_VOLUME='<temp-folder-to-be-used-for-storing-data>' \
+           -e LEANIX_HOST='<leanix-workspace-host>' \
+           -e LEANIX_REGION='<leanix-workspace-region>' \
            -e LEANIX_TOKEN='<leanix-technical-user-token>' \
+           -e GIT_PROVIDER='GITHUB' \
            -e GITHUB_TOKEN='<github-token>' \
            -e GITHUB_ORGANIZATION='<github-organization>' \
-           leanixacrpublic.azurecr.io/vsm-sbom-booster -rm
+           leanixacrpublic.azurecr.io/vsm-sbom-booster
 ```
+
+GitLab:
+```console
+docker run --pull=always --rm \
+           -v /var/run/docker.sock:/var/run/docker.sock \
+           -v <temp-folder-to-be-used-for-storing-data>:/tempDir \
+           -e MOUNTED_VOLUME='<temp-folder-to-be-used-for-storing-data>' \
+           -e LEANIX_HOST='<leanix-workspace-host>' \
+           -e LEANIX_REGION='<leanix-workspace-region>' \
+           -e LEANIX_TOKEN='<leanix-technical-user-token>' \
+           -e GIT_PROVIDER='GITLAB' \
+           -e GITLAB_TOKEN='<github-token>' \
+           -e GITLAB_GROUP='<github-organization>' \
+           leanixacrpublic.azurecr.io/vsm-sbom-booster
+```
+
 2. After a while your mapping inbox should be receiving new discovery items. These will need to be mapped by you once (see our [user documentation](https://docs-vsm.leanix.net/docs/discover-automate#create-your-service-baseline)).
 
 ### Environment Variables
 
+#### Runtime settings
 The first `-v` param is needed as the setup is a docker-in-docker setup and will need to mount the local docker runtime into the container. Under normal circumstances this param can be copied and pasted.
 
 The second `-v` param is the path to temporary folder that the `vsm-sbom-booster`will use to temporarily clone the projects to attempt to generate the SBOM. e.g. `~/output/temp`. Docker should have Read/Write access on this folder.
 
 `MOUNTED_VOLUME`: this is the same as the second `-v` param. It's required as the container needs an explicit env variable to do its job.
 
+`CONCURRENCY_FACTOR`(optional): the number of parallel jobs `vsm-sbom-booster` will use to generate SBOMs. Note: increasing this number will come at higher compute costs. Default: 3
+
+#### LeanIX configs
+
+`LEANIX_HOST`: The host name of your LeanIX workspace. e.g. For `https://acme.leanix.net` you would provide `acme`
+
+`LEANIX_REGION`: The region that your VSM workspace is hosted in. One of: eu|de|us|au|ca|ch
+
 `LEANIX_TOKEN`: API token with **ADMIN** rights from your VSM workspace. (see admin > technical users).
 
+#### Discovery API data
```

Comment:
good idea! may you add the link to the API docs pls?

Reply:
Can't believe I missed that, will add it for sure.